advocacy_docs/partner_docs/VeritasGuide/04-ConfiguringTheIntegration.mdx

```diff
@@ -0,0 +1,47 @@
+---
+title: 'Configuring the Integration'
+description: 'Walkthrough on configuring NetBackup for PostgreSQL'
+---
+
+
+### Prerequisites
+
+- A running EDB Postgres Advanced (EPAS)
+- A running NetBackup Server with a DataStore Policy set up
+- NetBackup Client installed
+- NetBackup PostgreSQL Agent installed
+
+
+### Configuring (Veritas NetBackup for PostgreSQL)
+
+The following steps show an example of how to configure NetBackup for PostgreSQL for EDB Postgres Advanced 13 on CentOS 7.
+
+!!! Note
+    For detailed information, refer to the [Veritas NetBackup for PostgreSQL Administrator's Guide](https://www.veritas.com/content/support/en_US/doc/129277259-150015228-0/v129277275-150015228).
+
+
+1.  Update configuration file /usr/NBPostgreSQLAgent/nbpgsql.conf to set the following parameters:
+    ```
+    DB_USER= enterprisedb
+    DB_PORT=5444
+    DB_INSTANCE_NAME=edb
+    PGSQL_LIB_INSTALL_PATH= /usr/edb/as13/lib
+    MASTER_SERVER_NAME=<veritas-master-server-name>
+    POLICY_NAME=<datastore-policy-name>
+    SCHEDULE_NAME= Default-Application-Backup
+    SNAPSHOT_SIZE=4MB
+    ```
+
+2.	Create the archivedir directory
```

Comment:
what do you mean my this?  what is it?  how and where do you create it?

Reply:
Archivedir is a standard configuration parameter for Postgres databases - it is where the user wants the database to be archived.